examples/cis/config.yaml

```diff
@@ -0,0 +1,4 @@
+write-kubeconfig-mode: 600
+cni: calico
+profile: "cis-1.23"
```

Comment:
the ` profile: cis ` is the newer syntax for this to avoid hitting deprecations

Reply:
fixed, thank you!